mlflow/pipelines/steps/evaluate.py

```diff
@@ -184,7 +185,7 @@ def my_warn(*args, **kwargs):
             ].greater_is_better
 
             _set_experiment_primary_metric(
-                exp_id, f"{self.primary_metric}_on_data_test", primary_metric_greater_is_better
+                exp_id, f"test_{self.primary_metric}", primary_metric_greater_is_better
```

Comment:
Just to confirm, this is the format used by Databricks AutoML, correct?

Reply:
Yes, Databricks AutoML uses the same format.